Makefile

```diff
@@ -0,0 +1,19 @@
+TARGETS = install|uninstall
+
+help:
+	echo "Usage: make <$(TARGETS)>"
+
+install:
+	install -m755 irssi-notifier.sh $$HOME/.local/bin/irssi-notifier.sh
```

Comment:
$HOME/.local/bin doesn't seem to be in $PATH here by default. Is there a way to make this more generic?